src/components/Cart/Cart.js

```diff
@@ -20,16 +21,28 @@ const Cart = ({ isCartVisible, closeCart, content, removeFromCart }) => {
             ? `${content.length} game`
             : `${content.length} games`}
         </div>
-        <div className={styles.x} onClick={closeCart}>
+        <div
+          className={styles.x}
```

Comment:
This should be renamed to something more intuitive smth like `closeButton` instead of `x`